lib/spack/spack/environment/environment.py

```diff
@@ -773,7 +773,7 @@ def _read_manifest(self, f, raw_yaml=None):
         )
         self.spec_lists[user_speclist_name] = user_specs
 
-        enable_view = config_dict(self.yaml).get("view")
+        enable_view = spack.config.get("view")
```

Comment:
@tgamblin @alalazo I changed this here to match moving the `view` section to a config section. In the write manifest section it looks at the env view and writes it out. I think that is fine..but maybe this needs to be re-thought now?